File: address-geocode-us-nodejs-examples/get_reverse_location_rest_sdk_example.js
import { GetReverseLocationClient } from "../address-geocode-us-nodejs/REST/get_reverse_location_rest.js";

async function GetReverseLocationClientGo(licenseKey, isLive) {
    console.log("\n--------------------------------------------------");
    console.log("Address Geocode US - GetReverseLocation - REST SDK");
    console.log("--------------------------------------------------");

    const latitude = "34.419120";
    const longitude = "-119.703421";
    const timeoutSeconds = 15;

    console.log("\n* Input *\n");
    console.log(`Latitude       : ${latitude}`);
    console.log(`Longitude      : ${longitude}`);
    console.log(`License Key    : ${licenseKey}`);
    console.log(`Is Live        : ${isLive}`);
    console.log(`Timeout Seconds: ${timeoutSeconds}`);

    try
    {
        const response = await GetReverseLocationClient.invoke(latitude, longitude, licenseKey, isLive, timeoutSeconds);
        console.log("\n* Reverse Location Info *\n");
        if (response)
        {
            console.log(`Address   : ${response.Address}`);
            console.log(`City      : ${response.City}`);
            console.log(`County    : ${response.County}`);
            console.log(`State     : ${response.State}`);
            console.log(`Zip       : ${response.Zip}`);
        }
        else
        {
            console.log("No location info found.");
        }

        if (response.Error) {
            console.log("\n* Error *\n");
            console.log(`Error Description: ${response.Error.Desc}`);
            console.log(`Error Number     : ${response.Error.Number}`);
            console.log(`Error Location   : ${response.Error.Location}`);
        }
    }
    catch (e)
    {
        console.log("\n* Error *\n");
        console.log(`Error Message: ${e.message}`);
    }
}

export { GetReverseLocationClientGo };

File: address-geocode-us-dot-net-examples/GetReverseLocationRestSdkExample.cs
using address_geocode_us_dot_net.REST;

namespace address_geocode_us_dot_net_examples
{
    public static class GetReverseLocationRestSdkExample
    {
        public static void Go(string licenseKey, bool isLive)
        {
            Console.WriteLine("\r\n--------------------------------------------------");
            Console.WriteLine("Address Geocode US - GetReverseLocation - REST SDK");
            Console.WriteLine("--------------------------------------------------");

            GetReverseLocationClient.GetReverseLocationInput getReverseLocationInput = new(
                Latitude: "34.419061",
                Longitude: "-119.702139",
                LicenseKey: licenseKey,
                IsLive: isLive,
                TimeoutSeconds: 15
            );

            Console.WriteLine("\r\n* Input *\r\n");
            Console.WriteLine($"Latitude    : {getReverseLocationInput.Latitude}");
            Console.WriteLine($"Longitude   : {getReverseLocationInput.Longitude}");
            Console.WriteLine($"License Key : {getReverseLocationInput.LicenseKey}");
            Console.WriteLine($"Is Live     : {getReverseLocationInput.IsLive}");

            GetReverseLocationResponse response = GetReverseLocationClient.Invoke(getReverseLocationInput);
            if (response.Error == null)
            {
                Console.WriteLine("\r\n* Reverse Location Info *\r\n");
                Console.WriteLine($"Address : {response.Address}");
                Console.WriteLine($"City    : {response.City}");
                Console.WriteLine($"County  : {response.County}");
                Console.WriteLine($"State   : {response.State}");
                Console.WriteLine($"Zip     : {response.Zip}");
            }
            else
            {
                Console.WriteLine("\r\n* Error *\r\n");
                Console.WriteLine("\r\n* Error *\r\n");
                Console.WriteLine($"Error Number  : {response.Error.Number}");
                Console.WriteLine($"Error Desc    : {response.Error.Desc}");
                Console.WriteLine($"Error Location: {response.Error.Location}");
            }
        }
    }
}